test/integration/document_locale_param_canonicalisation_test.rb

```diff
@@ -10,7 +10,7 @@ def with_locale_param(path, locale)
     u.to_s
   end
 
-  announcement_redir_document_types = %w(news_article speech)
```

Comment:
Should this part be deleted? My interpretation is that It relates to the index of speeches which hasn't changed?